src/main/java/de/leonhard/storage/annotation/Config.java

```diff
@@ -0,0 +1,12 @@
+package de.leonhard.storage.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.FIELD})
+public @interface Config {
```

Comment:
The name config clashes with the other "Config" class in the project. This is kind of confusing. We need a better name for this. "ConfigPath" seems way more accurate to me.

Reply:
Resolved in 81a473f